cloud/google/clients/computeservice.go

```diff
@@ -12,6 +25,19 @@ type ComputeService struct {
 	service *compute.Service
 }
 
+type GCEClientComputeService interface {
```

Comment:
Moved back to machineactuator. Since cluster and machine actuator are in the same package it only needs to be defined in one of them.

Reply:
Sorry to do this to you -- but if it's being used by both MachineActuator AND ClusterActuator (which I think is appropriate rather than duplicating the interface) then we should probably move the interface OUT of machineactuator.

Maybe another file in the 'google' package is needed and the interface can move into there? @krousey / @roberthbailey do you have any golang guidance on this scenario?